manifest.toml

```
@@ -1,67 +1,112 @@
-#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
-
+# https://yunohost.org/en/packaging_manifest
 packaging_format = 2
 
 id = "django-fmd"
 name = "django-fmd"
-description.en = "Web based FritzBox management using Python/Django."
+description.en = "YunoHost app package for https://gitlab.com/jedie/django-find-my-device"
```

Please describe directly what the app does ...

```suggestion
description.en = "Track, wipe and issue other commands to your device when it's lost"
```